src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,31 +36,47 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         if (!isValidAddress(trimmedAddress)) {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
-        this.value = trimmedAddress;
+
+        // Split into different parts of address
```

Comment:
This is a redundant comment. We can see from the code that you are splitting the address :wink: